### PR TITLE
Bootstrap assignment 2

### PR DESCRIPTION
**Pull request by meta-anshita-arya**

### CODE REVIEW

File: Web/BootStrap-Assignment2/BootstrapForm.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		<link rel="stylesheet" href="css/style.css" type="text/css">
+		<script type="text/javascript" src="js/bootstrap.min.js"></script>
+	</head>
+	<body>
+		<h2 class="hidden-xs">Sample Form</h2>
+		<h3 class="visible-xs "><span class="glyphicon glyphicon-film"></span></h3>
+		<div class="well">
+			<form class="form-horizontal" role="form">
+				<div class="row">
+					<div class="col-lg-1 col-md-2 col-sm-4 col-xs-12">Movie Title</div>
```
Comment (certinia2023) [inline Web/BootStrap-Assignment2/BootstrapForm.html:17]: Use other HTML 5 element also to design a layout : like label ,section, other

File: Web/BootStrap-Assignment2/BootstrapForm.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+		<link rel="stylesheet" href="css/bootstrap.min.css" type="text/css">
+		<link rel="stylesheet" href="css/style.css" type="text/css">
+		<script type="text/javascript" src="js/bootstrap.min.js"></script>
+	</head>
+	<body>
+		<h2 class="hidden-xs">Sample Form</h2>
+		<h3 class="visible-xs "><span class="glyphicon glyphicon-film"></span></h3>
+		<div class="well">
```
Comment (certinia2023) [inline Web/BootStrap-Assignment2/BootstrapForm.html:14]: Use container class

File: Web/BootStrap-Assignment2/BootstrapForm.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+					<div class="col-lg-1 col-md-2 col-sm-4 col-xs-12">Youtube trailer</div>
+					<div class="col-lg-5 col-md-4 col-sm-8 col-xs-12">
+						<input type="text" name="youtube" class="form-control">
+					</div>
+				</div>
+				<div class="row">
+					<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">Review</div>
+				</div>
+				<div class="row">
+					<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12"><input type="textarea" name="review" class="form-control textarea"></div>
```
Comment (pragati2015) [inline Web/BootStrap-Assignment2/BootstrapForm.html]: Us

File: Web/BootStrap-Assignment2/BoostrapMenu.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+       </li>
+	   </ul>
+		<ul class="nav navbar-nav navbar-right">
+		  <li>
+			<form class="navbar-form" role="search"> 
+				<div class="input-group"> 
+					<input type="text" class="form-control" placeholder="Search this site" name="srch-term" id="srch-term">
+					 <div class="input-group-btn">
+						 <button class="btn btn-default" type="submit">
+						 <i class="glyphicon glyphicon-search"></i>
```
Comment (pragati2015) [inline Web/BootStrap-Assignment2/BoostrapMenu.html]: 1.Search icon is not proper use some padding-top and bottom so that it's look like same height of text box
2. for small screen size devices  the placeholder of text box is not showing properly for small size device use small font of placeholder

File: Web/BootStrap-Assignment2/BoostrapMenu.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+       <li><a href="#">Home</a></li>
+				<li><a href="#">About</a></li>
+				<li><a href="#">Contact Us</a></li>
+				<li><a href="#">Logout</a></li>
+				<li><a href="#">Movie Review</a></li>
+				<li><a href="#">Movie Form</a></li>	
+				<li class="dropdown">
+					<a class="dropdown-toggle" data-toggle="dropdown" href="#">CSS<span class="caret"></span></a>
+						<ul class="dropdown-menu">
+							<li><a href="#">Button</a></li>
```
Comment (certinia2023) [inline Web/BootStrap-Assignment2/BoostrapMenu.html:89]: extra small devices table is not responsive

File: Web/BootStrap-Assignment2/Search.html [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+			<a class="navbar-brand" href="#">Metacube</a>
+		</div>
+		<div class="collapse navbar-collapse" id="example-navbar-collapse">
+			<ul class="nav navbar-nav">
+				<li class="active"><a href="#" class="visible-sm"><span
+						class="glyphicon glyphicon-home"></span></a><a href="Question3.html"
+					class="hidden-sm">Home</a></li>
+				<li><a href="#" class="visible-sm"><span
+						class="glyphicon glyphicon-user"></span></a><a href="#"
+					class="hidden-sm">About</a></li>
```
Comment (certinia2023) [inline Web/BootStrap-Assignment2/Search.html:31]: 1.Add more record on JSON and search then every time it's showing only one record. it should show all the matching record not one record 
2.result table is not responsive for extra small devices
